### PR TITLE
EES-2703 - GH community standards

### PR DESCRIPTION
**Pull request by luke-h1**
This PR: 
- Adds basic community standard checklist that Github recommends. This includes things such as bug report templates, PR templates etc.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/-bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      required: true
+  - type: textarea
+    attributes:
+      label: Describe the Bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected Behavior
```
Comment (sarahmwong) [inline .github/ISSUE_TEMPLATE/-bug_report.yml:40]: Behavior  <- "Behaviour"

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+### Our Standards
+
+Examples of behavior that contributes to creating a positive environment
```
Comment (sarahmwong) [inline CODE_OF_CONDUCT.md:14]: Behavior  <- "Behaviour"

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+- Using welcoming and inclusive language
+- Being respectful of differing viewpoints and experiences
+- Gracefully accepting constructive criticism
+- Focusing on what is best for the community
+- Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery and unwelcome sexual attention or
```
Comment (sarahmwong) [inline CODE_OF_CONDUCT.md:25]: sexualized <- "sexualised"

**Review by sarahmwong — COMMENTED** [review]
Looks good - some Americanisms (only spotted because Cam had flagged these in one I did the other week for one of our repos!) just need to do a find and replace for these in the files I've added comments to

File: .github/ISSUE_TEMPLATE/-bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+  - type: dropdown
+    id: packageManager
+    attributes:
+      multiple: true
+      label: What package manager are you using / does the bug impact?
+      options:
+        - npm
+        - yarn 
+        - pnpm
```
Comment (ntsim) [inline .github/ISSUE_TEMPLATE/-bug_report.yml:17]: Not sure it's worth putting this in. They should only be using whatever is recommended by the repo i.e. NPM.

File: .github/ISSUE_TEMPLATE/-bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    attributes:
+      label: What operating system are you using?
+      options:
+        - Mac
+        - Windows
+        - Linux
+        - Other
+    validations:
+      required: true
+  - type: textarea
```
Comment (ntsim) [inline .github/ISSUE_TEMPLATE/-bug_report.yml:32]: Should probably add a question for the browser they were using

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<!--
+  Thanks for submitting a pull request!
+  We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request.
+  
+  Before submitting a pull request, please make sure the following is done:
+
+  1. Clone [the repository](https://github.com/dfe-analytical-services/explore-education-statistics) and create your branch from `dev`.
+  2. Run `npm ci && npm run bootstrap` in the repository root.
+  3. If you've fixed a bug or added code that should be tested, add tests!
+  4. Ensure the test suites passes (`npm run test`).
```
Comment (ntsim) [inline .github/pull_request_template.md:10]: Would suggest adding stuff about the following too:
- Backend tests should be run using `dotnet test` from `src`
- UI tests should be run from `tests/robot-tests`

File: .github/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+-->
+
+## Summary
+
+<!--
+ Explain the **motivation** for making this change. What existing problem does the pull request solve?
+-->
+
```
Comment (ntsim) [inline .github/pull_request_template.md:18]: Could have other sections with heading/descriptions like:

- Related changes - Any changes that were closely related to this change but not directly part of the main work e.g. refactoring, fixes, etc.
- Screenshots - Add some screenshots if it helps illustrate the change made

**Review by cjrace — APPROVED** [review]

**Pull request merged**